zcash_primitives/src/sapling/builder.rs

```diff
@@ -154,7 +143,7 @@ impl SpendDescriptionInfo {
 /// A struct containing the information required in order to construct a
 /// Sapling output to a transaction.
 #[derive(Clone)]
-struct SaplingOutputInfo {
+pub struct SaplingOutputInfo {
```

Comment:
nit: I'd love to come up with better names for these before making them public. The `Info` suffix is just meaningless.